### ISSUE TITLE
Can the design drawing of the car provide a SolidWorks version

### ISSUE THREAD

**Issue by bluesky-ryan** [jakkra/Mars-Rover#23]
Dear jakkra:

   Thank you for your selfless sharing. Your rover is very cool. I also want to make one, but Fusion360 is very slow in China, and the license is basically not available. Is there a solidworks version of the car? Whether the bearings on the 4 driving wheels are missing, I wondered how to install them for a long time.

**Comment by jakkra** [comment]
Hi, unfortunately from what I know that's not possible. What is possible is to export the Rover in .step format, I think Solidworks should be able to handle that. 
From my quick google search Fusion 360 should be fine to use in china and the free hobby license also valid in china. However I tried to export the whole rover as .step so you can try to import that, that's the best I can do. [rover_full.zip](https://github.com/jakkra/Mars-Rover/files/5164127/rover_full.zip)

**Comment by jakkra** [comment]
For the bearing see this screenshot:
![steering_bearing_mounting](https://user-images.githubusercontent.com/4318648/92022843-2f9fd700-ed5c-11ea-968a-c4c09531700d.JPG)

**Comment by bluesky-ryan** [comment]
thank you very much!!! love you

**Comment by bluesky-ryan** [comment]
Dear jakkra:

    What are the dimensions of these joints? I can’t draw these joints without dimensions.

![image](https://user-images.githubusercontent.com/26185987/92497383-db996480-f22b-11ea-9e01-1adb9e9cb1f5.png)

**Comment by bluesky-ryan** [comment]
This one I painted is very ugly, and the size is not right at all

![image](https://user-images.githubusercontent.com/26185987/92497631-2b782b80-f22c-11ea-85ec-d3d2a3d60f88.png)

**Comment by jakkra** [comment]
I thought dimentions was included in the .step files. Are you sure you cannot use some measure tool in Soldiworks?

**Comment by bluesky-ryan** [comment]
I tried the measuring tool inside, but this kind of irregular curved surface cannot be measured.，very sad. Is it necessary to use the space triangle Pythagorean theorem to calculate the size, mainly to keep the 3 wheels in a straight line and the middle wheel is exactly in the middle?

**Comment by jakkra** [comment]
The middle wheel is not in the exact middle. Are you re-drawing the rover in Solidworks? I don't really follow. I can take some measurements if you specify exactly which distances you need.

**Comment by bluesky-ryan** [comment]
Yes, on the one hand, some parts have different sizes and need to be redrawn. On the other hand, I also hope to be able to draw pictures by myself like you. I am learning to draw. You are now my idol. The design of the wheel support frame of your car is very ingenious. I have watched it for 2 days and cannot understand your design idea. How do you ensure that the wheels are in the same straight line? The support rods and support points of the rear wheels are not in the same line. This design in a straight line feels very cool, but I can’t understand how you can ensure the exact dimensions.

**Comment by jakkra** [comment]
It's not that hi-tech, I just modeled from the ground up:
Created 1 wheel and wheel base, copied it 2 times and put them in a straight line, now we have 3 wheels in a straight line.
Then model the pipe in a direction so it meets the other pipes (some trial and error needed). Put a large square that connects the pipes coming from each wheel and TADA.

**Comment by bluesky-ryan** [comment]
lol..   I thought you designed this frame according to a certain kinematics principle, doing complex calculations, and then started drawing parts according to this frame

**Comment by Smarty-pro** [comment]
@bluesky-ryan this is what NASA did, this rover is an imitation of the famous Curiosity rover
